### ISSUE TITLE
Should the examples inside the `examples` folder be moved to the repo `ndarray-examples`?

### ISSUE THREAD

**Issue by nbro** [rust-ndarray/ndarray#1254]
It seems that the examples here https://github.com/rust-ndarray/ndarray/tree/master/examples can be moved to https://github.com/rust-ndarray/ndarray-examples, which, in my view, doesn't just have to contain machine learning examples, but could contain even simple examples to help newbies use `ndarray`.

**Comment by bluss** [comment]
Yes, and some less useful examples can be removed. Confession, code like examples/sort-axis.rs is more like prototype code for later inclusion in ndarray (hasn't happened yet), so it should probably stay around or be moved to a PR.
